telco-core/configuration/reference-crs/required/networking/tuned-queues.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: configuration-nic
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Configuration to set to 32 the nb of interrupt per NIC
+      include=openshift-node-performance-$name
```

Comment:
In the reference configuration we also include the PerformanceProfile. We should explicitly align the names here along with a note (comment) that if you change one you need to change the other.

Reply:
Yep, we should, but the included profile also uses $name here https://github.com/openshift-kni/telco-reference/blob/main/telco-core/configuration/reference-crs/required/performance/PerformanceProfile.yaml#L7

Reply:
I could use lookupCR here to match the perf profile name, however it would not work when multiple profiles (and multiple MCPs) show up..